src/tools/unpack.c

```diff
@@ -976,15 +984,32 @@ static void foreach_unpack_entry(struct UnpackOptions *opts, void (*f)(struct Un
     for (int i=0; i<opts->header.entries_length; i++) {
         struct UnpackFileEntry *e = &(opts->fentries[i]);
 
+        if (opts->abort) {
+            SFMF_FAIL("Operation aborted via D-Bus\n");
+        }
```

Comment:
Other places in code seem to assume SFMF_FAIL returns. Should there be a break/return/something here?